content/authentication/connecting-to-github-with-ssh/testing-your-ssh-connection.md

```diff
@@ -36,7 +36,7 @@ When you test your connection, you'll need to authenticate this action using you
   > Are you sure you want to continue connecting (yes/no)?
   ```
```

Comment:
- [x] ###__

Reply:
> - [x] ###__

R/%##**}

Reply:
Sorry. I can't understand what you are trying to convey.

Reply:
D//###